spec/rubocop/cop/rails/delegate_spec.rb

```diff
@@ -353,4 +353,22 @@ def foo
       delegate :foo, to: :$global_variable
     RUBY
   end
+
+  it 'is disabled for controllers' do
+    expect_no_offenses(<<~RUBY, "#{Bundler.root}/app/controllers/foo_controller.rb")
+      class FooController
+        before_action :load
+
+        def destroy
+          @foo.destroy
+        end
+
+        private
+
+        def load
+          @foo = Foo.find(params[:id])
+        end
```

Comment:
seems like an overly complex example, no? the `def destroy` piece should suffice

Reply:
I also think this is good as test code for documentation.